content/body/marquee.php

```diff
@@ -1,5 +1,5 @@
 <p>
-    Marquees are meant for content that scrolls or updates consistently, like a stock ticker or a news feed. Although the have a default <code>aria-live</code> value of <code>off</code>, you can use <code>aria-live="polite"</code> to let users hear the information within a marquee in almost real-time.  Do this carefully, the last thing you would want is have a screen reader update too much in a way that would make the rest of your application unusable to screen reader users due to too much screen reader noise.
+    Marquees are meant for content that scrolls or updates consistently, like a stock ticker or a news feed. Although they have a default <code>aria-live</code> value of <code>off</code>, you can use <code>aria-live="polite"</code> to let users hear the information within a marquee in almost real-time.  Do this carefully, the last thing you would want to have a screen reader update too much in a way that would make the rest of your application unusable to screen reader users due to too much screen reader noise.
```

Comment:
While adding the "to" word makes sense, it still needs the "is" word.
```suggestion
    Marquees are meant for content that scrolls or updates consistently, like a stock ticker or a news feed. Although they have a default <code>aria-live</code> value of <code>off</code>, you can use <code>aria-live="polite"</code> to let users hear the information within a marquee in almost real-time.  Do this carefully, the last thing you would want is to have a screen reader update too much in a way that would make the rest of your application unusable to screen reader users due to too much screen reader noise.
```